### PR TITLE
Update DG to personalize to our app

### LINKED ISSUE #15: Update DG: Target user profile, value proposition, and user stories
- Update the target user profile and value proposition to match the project direction.
- Give a list of the user stories (and update/delete existing ones, if applicable), including priorities.

### PR DESCRIPTION
**Pull request by craigtonlian**
Closes #15

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,14 +3,14 @@
   title: ""
 ---
 
-# AddressBook Level-3
+# Staff-Snap
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```
Comment (CelestineTan03) [inline docs/index.md:8]: This is the link for AB3; maybe can update it to our project? 😄

**Review by CelestineTan03 — CHANGES_REQUESTED** [review]
Just update the ci and codecov badge in index.md then will be good to merge 😄

File: docs/index.md (hunk 1 above)
Comment (craigtonlian) [reply docs/index.md]: Sounds good! 🐶

**Review by CelestineTan03 — APPROVED** [review]
Lgtm! 🐻

**Pull request merged**